src/how_to/stabilize.md

```diff
@@ -0,0 +1,20 @@
+# Stabilize a feature
+
+The final step in the [language-change process](./propose.md) is to **stabilize** a feature. Stabilization works as follows:
+
+* Author a [stabilization report][sg]:
+    * Briefly recap the feature's design from the RFC -- you don't have to go into detail, we can re-read the RFC.
+    * Give detailed descriptions of how the feature's design has changed since the RFC was approved!
+    * Summarize any major decisions that were made during the implementation process.
+    * Verify that the feature is fully implemented. Look for tests covering all the major pieces of the RFC and include them in the stabilization report.
+    * Provide answers to any "unresolved questions" listed in the RFC.
+    * Describe the implementation history of the feature (optional).
+* Prior to stabilizing, we need to coordinate with other teams:
+    * An open PR editing the reference to describe the change is required. (You don't personally have to author it, but there needs to be an open PR, and ideally one that has been edited and is generally accepted.)
+    * If the feature affects the type system, you should ping @rust-lang/types to check for their approval.
+    * If the feature adds new syntax, you should ping the style team.
```

Comment:
```suggestion
    * If the feature affects the type system, you should nominate for @rust-lang/types to check for their approval.
    * If the feature adds new syntax, you should should nominate for the style team to get their approval.
```

In general, it feels like "ping" is an easy stand-in for "shout into void"; I think we ought to use the mechanism we have for getting a team's attention which is nomination. Much more durable and less noisy.

Reply:
That makes sense.